static/docker-compose.yml

```diff
@@ -27,17 +27,32 @@ services:
     - plextracdb
     - redis
     image: "plextrac/plextracapi:${UPGRADE_STRATEGY:-stable}"
+    volumes:
+    - uploads:/usr/src/plextrac-api/uploads:rw
     entrypoint: ""
     command: |
       sh -c
         "npm run maintenance:enable &&
          npm run db:migrate &&
+         npm run pg:migrate &&
+         npm run pg:etl up all &&
          npm run maintenance:disable"
     environment:
+      COUCHBASE_URL: "http://plextracdb"
```

Comment:
I guess the `db:migrate` script was applying a default value for this. But our ETL script is not.